src/main/scala/com/nawforce/commands/DependencyExplorer.scala

```diff
@@ -218,12 +218,12 @@ class DependencyExplorer(context: ExtensionContext, server: Server) {
       try {
         val ignorePattern = Pattern.compile(ignoreTypes.getOrElse("^$"))
         graph.nodeData.toIndexedSeq.zipWithIndex.flatMap(nd => {
-          val typeName = nd._1.identifier.typeName.toString()
+          val typeIdentifier = nd._1.identifier.toString()
```

Comment:
This is for consistency, the strings used in the web view are generated from a TypeIdentifier rather than TypeName.